tensorflow/core/kernels/ragged_tensor_to_tensor_op.cc

```diff
@@ -138,8 +138,11 @@ class RaggedTensorToTensorBaseOp : public OpKernel {
     return std::max(index_length - first_equal_index, max_width);
   }
 
-  absl::Status CalculateOutputSize(INDEX_TYPE first_dim, OpKernelContext* c,
+absl::Status CalculateOutputSize(INDEX_TYPE first_dim, OpKernelContext* c,
```

Comment:
```suggestion
 absl::Status CalculateOutputSize(INDEX_TYPE first_dim, OpKernelContext* c,
```

Please add back the space that you removed.